front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -163,14 +163,18 @@ const ManchetteWithSpaceTimeChartWrapper = ({
   return (
     <div className="manchette-space-time-chart-wrapper">
       <div className="header">
-        <ManchetteMenuButton setWaypointsPanelIsOpen={setWaypointsPanelIsOpen} />
-        {waypointsPanelIsOpen && waypointsPanelData && (
-          <WaypointsPanel
-            waypointsPanelIsOpen={waypointsPanelIsOpen}
-            setWaypointsPanelIsOpen={setWaypointsPanelIsOpen}
-            waypoints={operationalPoints}
-            waypointsPanelData={waypointsPanelData}
-          />
+        {waypointsPanelData && (
+          <>
+            <ManchetteMenuButton setWaypointsPanelIsOpen={setWaypointsPanelIsOpen} />
+            {waypointsPanelIsOpen && waypointsPanelData && (
```

Comment:
Nit: `waypointsPanelData` is already checked to be non-undefined above, so we don't need to check it again in the nested condition.

Reply:
my bad, brain still confused from the past three days ! thanks !